modules/33acrossIdSystem.js

```diff
@@ -39,9 +39,11 @@ function calculateResponseObj(response) {
   };
 }
 
-function calculateQueryStringParams(pid, gdprConsentData) {
+function calculateQueryStringParams(pid, consentData) {
```

Comment:
Why the change in param name given that it still is used only in gdpr check? It is also invoked in ajax call as `gdprConsentData` https://github.com/33Across/Prebid.js/blob/c4314a0ed58cc4f90891057dde8fceb5a802b0b4/modules/33acrossIdSystem.js#L138